plopfile.js

```diff
@@ -61,28 +57,48 @@ const customHookGenerator = () => ({
         templateFile: 'plop-templates/hook/hook.index.hbs',
       },
     ];
-
-    return actions;
   },
 });
 
 const reactComponentGenerator = () => ({
   description: componentTypes.REACT_COMPONENT,
   prompts: [
-    {
-      type: 'list',
-      name: 'baseDir',
-      message: 'base directory',
-      choices: ['ui', 'app'],
-      default: 1,
-    },
     {
       type: 'directory',
       name: 'directory',
       message: 'select directory',
-      basePath: './src/ui',
-      when: answers => answers.baseDir === 'ui',
+      basePath: './src/app',
+    },
+    {
+      type: 'input',
+      name: 'name',
+      message: 'component name',
     },
+  ],
+  actions: function() {
+    return [
+      {
+        type: 'add',
+        path: `src/app/{{directory}}/{{camelCase name}}/{{pascalCase name}}.tsx`,
+        templateFile: 'plop-templates/component/Component.hbs',
+      },
+      {
+        type: 'add',
+        path: `src/app/{{directory}}/{{camelCase name}}/{{pascalCase name}}.test.tsx`,
+        templateFile: 'plop-templates/component/Component.test.hbs',
+      },
+      {
+        type: 'add',
+        path: `src/app/{{directory}}/{{camelCase name}}/{{pascalCase name}}.types.ts`,
+        templateFile: 'plop-templates/component/Component.types.hbs',
+      },
+    ];
+  },
+});
+
+const reactContainerComponentGenerator = () => ({
```

Comment:
Currently, only the top level of directories in the app dir is being listed, maybe we could change that by using a custom inquirer prompt like this one? (plop uses inquirer) https://github.com/anc95/inquirer-file-tree-selection

Reply:
my bad works fine